frontend-react/src/components/FileHandlers/FileHandlerMessaging.tsx

```diff
@@ -140,14 +137,6 @@ export const RequestedChangesDisplay = ({
         data.length &&
         data.some((responseItem) => responseItem.message);
 
-    useEffect(() => {
```

Comment:
Two things:
- We don't need to log out these errors anymore because that'll get caught by PagerDuty unnecessarily.  Fixes #8483 
- If we wanted to do this, this isn't really something that should be done in an effect; it should be done when we get the response.